config.json

```diff
@@ -61,6 +61,13 @@
       ]
     },
     {
+      "slug": "acronym",
+      "difficulty": 2,
+      "topics": [
+        "String Manipulation"
```

Comment:
> breadth of difficulty
> Perhaps there are more underlying topics that we haven't discovered yet, but currently they can't rely only on `String` to be ordered.

Indeed, maybe we will have to think of a more precise topic.

> That said, do we have a clear/fixed way to order the exercises based on their topics?

I can't really say the current system is clear at all, really. Surely the hardest topic is important, but there's no mapping of topics to numbers nor is there a rule for whether any additional topics would or would not make an exercise harder. They're just guesses. They could be wrong.